client/verta/tests/test_concurrency/run_functions.py

```diff
@@ -0,0 +1,30 @@
+import random
+import pytest
+import os
+
+
+def log_observation(client, i):
+    observations = []
+    run = client.set_experiment_run()
+
+    for _ in range(5):
+        observations.append(random.random())
+        run.log_observation("obs", observations[-1])
+
+    return {"run": run, "obs": observations}
+
+
+def upload_artifact(client, i):
+    run = client.set_experiment_run()
+
+    PART_SIZE = int(5.4 * (10 ** 6))  # 5.4 MB; S3 parts must be > 5 MB
+    os.environ['VERTA_ARTIFACT_PART_SIZE'] = str(PART_SIZE)
```

Comment:
I don't think it would, but this doesn't affect the environment variables in the parent process, right? Just to make sure it doesn't affect subsequent tests.

Reply:
Doesn't seem so. I tried printing out os.environ.get("VERTA_ARTIFACT_PART_SIZE"), and it returns None. This conforms to my understanding of processes in general (i.e not sharing variables between process).